### DISCUSSION TITLE
Emgu.CV.Platform.IOS.dll size

### DISCUSSION THREAD

**Discussion by milanm** [emgucv/emgucv#699] [Q&A] [answered]
I just checked my bin/Debug folder and see that Emgu.CV.Platform.IOS.dll size is about 654Mb!! Its too big to be deployed to an IOS device. On Android the whole apk increased in size for about 170Mb when Emgu.CV was added.

**Comment by milanm** [comment, marked as answer]
I found the solution: https://www.emgu.com/wiki/index.php/Download_And_Installation#Adding_Emgu_CV_to_your_Xamarin_iOS_App_using_nuget_package

**Comment by emgucv** [comment]
Right, there are detail explations in the link above for the size of the DLL, it is 645MB during development before any stripping is applied. 

The final size in the IPA for App Store release for a single architecture (e.g. ARM64) should be ~40MB for Emgu CV alone.
